src/Signer.php

```diff
@@ -31,14 +34,40 @@ public function __construct($key, $algorithm, $headerList)
      * @param RequestInterface $message
      * @return RequestInterface
      */
-    public function sign($message)
+    public function sign($message, $withDigest = false)
```

Comment:
Would a `signWithDigest` method be better here instead of adding an argument

Reply:
Fixed in b5c6d788b50ceb24d142e5f56df287fe1816680e